src/services/user.service.js

```diff
@@ -0,0 +1,33 @@
+const { User } = require('../models/User.model');
+const jwtService = require('../services/jwt.service');
+
+const getByActivationToken = async (activationToken) => {
+  return User.findOne({ where: { activationToken } });
+};
+
+const getByEmail = async (email) => {
+  return User.findOne({ where: { email } });
+};
+
+const getById = async (id) => {
+  return User.findOne({ where: { id } });
+};
+
+const getByAccessToken = async (accessToken) => {
+  const { id } = await jwtService.verify(accessToken);
```

Comment:
Consider adding error handling for the 'jwtService.verify' call. If the access token is invalid or expired, this will throw an error, which should be handled appropriately.